R/utils.R

```diff
@@ -223,6 +223,10 @@ pruneViewportAdjustmentFromBins <- function(bins, xVP, x, viewport) {
 }
 
 findBinStart <- function(x) {
+  if (!length(x)) {
```

Comment:
The findBinStart and end functions return what we'd expect for x all NAs as well?

Reply:
trouble w doing some of the work months ago is i cant quite remember this one. think im trying to make sure the data object is valid in structure but empty here. but proves your other point about tests for the data structure...